CHANGELOG.md

```diff
@@ -43,6 +43,12 @@
 
 # `libp2p` facade crate
 
+# 0.45.1 [unreleased]
+- Semver bump Rust from `1.56.1` to `1.60.0` . See [PR 2646].
```

Comment:
I am not sure if we have a written policy here but I think a minor version bump would be appropriate for a MSRV increase.

Reply:
I'm not able to find a written policy either but I would agree with you based on [Rust's  API Evolution's RFC](https://github.com/rust-lang/rfcs/blob/master/text/1105-api-evolution.md#principles-of-the-policy)

_In principle, it should be possible to produce a version of dependency code that will not break when upgrading other dependencies, or Rust itself, to a new minor revision._